### ISSUE TITLE
How can I access the docs in production

### ISSUE THREAD

**Issue by joshajiniran** [knuckleswtf/scribe#694]
### Scribe version

4.22.0

### Your question

I have deployed my Laravel API on fly.io and I can't figure out how to access the docs with https://fly-app.fly.dev/docs for example.

I have used the type => static at first and it was a 404.

also used type => laravel and tried setting up a Route in web.php to view('scribe.index'), but got a 500, upon looking at the logs, here was the cause, since there isn't a scribe.postman and scribe.openapi view.
```
<ul class="toc-footer" id="toc-footer">
                    <li style="padding-bottom: 5px;"><a href="{{ route("scribe.postman") }}">View Postman collection</a></li>
                            <li style="padding-bottom: 5px;"><a href="{{ route("scribe.openapi") }}">View OpenAPI spec</a></li>
                <li><a href="http://github.com/knuckleswtf/scribe">Documentation powered by Scribe ✍</a></li>
</ul>
```
Any help would be appreciated

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by a4anthony** [comment]
@joshajiniran For a quick fix, I moved "knuckleswtf/scribe": "^4.19" from "require-dev" to "require" then run composer update and then deploy again.

**Comment by joshajiniran** [comment]
@a4anthony 

I really appreciate it 🙏

Works.
